object_store/src/gcp/credential.rs

```diff
@@ -195,6 +218,88 @@ impl OAuthProvider {
     }
 }
 
+fn reader_credentials_file<T>(
```

Comment:
```suggestion
fn read_credentials_file<T>(
```
Appreciate this was copied, but could take the opportunity to fix the name